app/views/submissions/show.html.erb

```diff
@@ -107,7 +107,7 @@
             <td></td>
           </tr>
           <% if ![:pending,:cancelled].include?(case_data.status) && (policy(@submission).inspect? || is_sample_set) %>
-            <tr class="inspect_test status_<%= case_data.status %>">
+            <tr class="inspect_test status_<%= case_data.status %><%= " sample_case" if is_sample_set %>">
```

Comment:
I feel like the 'sample_case' class should logically be on the first <tr> element (that also has the 'test_case' class), rather than on the second <tr> element.